projects/distributed-tracing/src/pages/trace-detail/trace-detail.page.component.ts

```diff
@@ -43,6 +43,13 @@ import { TraceDetails, TraceDetailService } from './trace-detail.service';
           <div class="separation"></div>
 
           <ht-copy-shareable-link-to-clipboard class="share"></ht-copy-shareable-link-to-clipboard>
+
+          <ht-download-json
+            class="download"
+            [dataSource]="this.exportSpans$"
+            [fileName]="traceDetails.id + '.json'"
```

Comment:
nit (slightly more readable, IMO): 
```suggestion
            fileName="{{traceDetails.id}}.json"
```

Reply:
Done!!